ADDING_AN_EXAMPLE.md

```diff
@@ -2,68 +2,39 @@
 
 > For ICP Ninja: check [NINJA_CONTRIBUTING.md](./NINJA_CONTRIBUTING.md) for how to contribute a project to ICP Ninja.
 
-Ideally, each example should come with both Rust and Motoko implementations,
-implementing the same Candid interface (and, ideally, semantics).
+Each example should be available in both Rust and Motoko variations, implementing the same Candid interface (and, ideally, semantics).
 
-To illustrate the pattern, this repo now contains one such example, project `hello`.
+To illustrate the pattern, this repo now contains one such example, project `hello_world`:
 
-The (dual) sources for the projects live in:
+`motoko/hello_world`
```

Comment:
the motoko examples is still archived
https://github.com/dfinity/examples/tree/master/archive/motoko/hello